src/auth/passport.js

```diff
@@ -1,24 +1,34 @@
 import passport from 'passport'
-import {Strategy as PassportJWTStrategy} from 'passport-jwt';
 import { Strategy as StrategyGoogle } from 'passport-google-oauth20'
-import { async } from 'regenerator-runtime'
-import { getOneOrCreateByGoogleDetails, getUserById } from '../services/user.service'
+import { getOneOrCreateByGoogleDetails } from '../services/user.service'
+import config from '../libs/config'
 
 export function getConfiguredPassport(){
+    /*
```

Comment:
we might want to further clean out all this old commented out code.

Reply:
@ryanolee ok